webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/resources/ProductPageResource.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership.  The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.oodt.cas.product.jaxrs.resources;
+
+import org.apache.oodt.cas.filemgr.structs.Product;
+import org.apache.oodt.cas.filemgr.structs.ProductPage;
+import org.apache.oodt.cas.filemgr.structs.Reference;
+import org.apache.oodt.cas.metadata.Metadata;
+
+import javax.xml.bind.annotation.*;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A JAX-RS resource representing a {@link ProductPage}.
+ * @author ngimhana (Nadeeshan Gimhana)
+ */
+@XmlRootElement(name="productPage")
+@XmlType(propOrder = {"pageSize","pageNum","totalPages","numOfHits","metadataResource", "productResources"})
+@XmlAccessorType(XmlAccessType.NONE)
+public class ProductPageResource
+{
```

Comment:
Formatting should be consistent. You can import Google Java format to IDEA from [this file](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml).